packages/app/src/components/Sidebar/Bookmarks.tsx

```diff
@@ -19,15 +20,36 @@ const BookmarksItem = ({ data }) => {
   const { page } = data;
   const dPagePath = new DevidedPagePath(page.path, false, true);
   const linkedPagePathLatter = new LinkedPagePath(dPagePath.latter);
-
+  const bookmarkItemId = `bookmark-item-${data._id}`;
   return (
-    <li className="list-group-item py-3 px-0" id={`bookmark-item-${data._id}`}>
-      <div className="d-flex w-100">
-        <h5 className="my-0">
-          <PagePathHierarchicalLink linkedPagePath={linkedPagePathLatter} basePath={dPagePath.isRoot ? undefined : dPagePath.former} />
-        </h5>
-      </div>
-    </li>
+    <>
+      <li className="list-group-item py-3 px-0" id={bookmarkItemId}>
+        <div className="d-flex w-100 justify-content-between">
+          <h5 className="my-0">
+            <PagePathHierarchicalLink linkedPagePath={linkedPagePathLatter} basePath={dPagePath.isRoot ? undefined : dPagePath.former} />
```

Comment:
## Tooltip
- `/test/1` 👉🏻 Tooltip should be  `/test/`
- `/test` 👉🏻 Tooltip should be `/`